lib/auth/auth.go

```diff
@@ -673,11 +673,32 @@ func (a *AuthServer) ValidateOIDCAuthCallback(q url.Values) (*OIDCAuthResponse,
 		return response, nil
 	}
 
-	user, err := a.Identity.GetUserByOIDCIdentity(services.OIDCIdentity{
+	var user services.User
+
+	user, err = a.Identity.GetUserByOIDCIdentity(services.OIDCIdentity{
```

Comment:
I think this change is out of scope of this PR and should be submitted separately. So please revert this commit from this PR.

 Can you also write down a small issue/proposal explaining the desired change so we can discuss it separately.